src/extractor/extraction_containers.cpp

```diff
@@ -136,6 +136,29 @@ void ExtractionContainers::FlushVectors()
     way_start_end_id_list.flush();
 }
 
+void ExtractionContainers::prepareCache()
+{
+    nodeCache.reserve(all_nodes_list.size());
+    for(int i = 0; i < all_nodes_list.size(); i++)
```

Comment:
`int` will overflow for the planet (and even smaller extracts) - make counter `size_t`